.github/workflows/dependency-updates.yml

```
@@ -0,0 +1,157 @@
+        run: |
+          echo "Updating dependencies..."
+          pnpm update --latest || echo "Dependency update completed with warnings"
+
+      - name: Check for changes
+        id: changes
+        run: |
+          git diff --exit-code pnpm-lock.yaml || echo "changes=true" >> $GITHUB_OUTPUT
+
+      - name: Create Pull Request
+        if: steps.changes.outputs.changes == 'true'
+        uses: peter-evans/create-pull-request@v5
```

actionlint (1.7.7)

the runner of "peter-evans/create-pull-request@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)